tests/Integration/StatementsCrudTest.php

```diff
@@ -0,0 +1,276 @@
+<?php
+require_once __DIR__ . '/../../vendor/autoload.php';
+
+use Doctrine\DBAL\DBALException;
+use Doctrine\DBAL\DriverManager;
+use Doctrine\DBAL\Exception\InvalidArgumentException;
+use Google\Cloud\Core\Exception\BadRequestException;
+use Oat\DbalSpanner\SpannerConnection;
+use Oat\DbalSpanner\SpannerDriver;
+use Oat\DbalSpanner\SpannerPlatform;
+
+class StatementsCrudTest extends \PHPUnit\Framework\TestCase
+{
+    protected const INSTANCE_NAME = 'php-dbal-tests';
+    protected const DATABASE_NAME = 'spanner-test';
+
+    /** @var SpannerConnection */
+    protected $connection;
+
+    /** @var int */
+    protected $now = 1558344254;
+
+    public function setUp(): void
+    {
+        $connectionParams = [
+            'dbname' => self::DATABASE_NAME,
+            'instance' => self::INSTANCE_NAME,
+            'driverClass' => SpannerDriver::class,
+            'wrapperClass' => SpannerConnection::class,
+            'platform' => new SpannerPlatform(),
+        ];
+
+        try {
+            $this->connection = DriverManager::getConnection($connectionParams);
+            $this->connection->connect();
+        } catch (\Exception $exception) {
+            var_dump('Unable to connect to Spanner instance. Did you forget to start an instance and setup a database prior to running the integration tests?');
```

Comment:
this is dicussable, but I feel that we need tests:
- that runs against a real spanner, totally agree
- but also tests that mocks spanner client, to be able to put more safely this lib on the CI pipeline